tenseal/sealapi/sealapi_util_namespace.cpp

```diff
@@ -716,6 +505,12 @@ void bind_util_namespace(pybind11::module &m) {
      * } "util/hash.h"
      *******************/
 
+    py::class_<MultiplyUIntModOperand>(m, "MultiplyUIntModOperand",
```

Comment:
can we have some sanity (py)tests for this class?

Reply:
Add a `test_util_multiplyuintmodoperand_sanity` test case in test_util.py